queries.sql

```diff
@@ -1,11 +1,18 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-ELECT * FROM animals WHERE name like '%mon';
+SELECT * FROM animals WHERE name like '%mon';
 SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
 SELECT * FROM animals WHERE neutered = true and escape_attempts < 3;
 SELECT name, date_of_birth FROM animals WHERE name = 'Agumon' or name = 'Pikachu';
 SELECT name, escape_attempts FROM animals WHERE weight_kg < 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts= 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT neutered, SUM(escape_attempts) AS "Who escaped the most?" FROM animals GROUP BY neutered;
```

Comment:
- [ ] You should be looking for `AVG (escape_attempts)` here 

```sql
SELECT neutered, AVG(escape_attempts) FROM animals
   GROUP BY neutered;
```

or you can keep `SUM(escape_attempts)` and add one more query  for AVG 😉